src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java

```diff
@@ -1,11 +1,14 @@
 package frc.robot.subsystems.elevator;
 
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
 import edu.wpi.first.math.trajectory.TrapezoidProfile;
 import edu.wpi.first.units.*;
 import frc.robot.Constants;
 import lib.webconstants.LoggedTunableNumber;
 
 public class ElevatorConstants { // TODO: check real values
+    public static final TalonFXConfiguration MOTOR_CONFIGURATION = new TalonFXConfiguration();
```

Comment:
Remember to configure everything (PID, current limits, inversion).

Reply:
\+ limit switch configurations.

Reply:
633215af3b67032937a55f35c09656598041f034

Reply:
7b9a422b712b2368f7b87172d5f9503acce4b913

Reply:
+ Motion Magic Expo

Reply:
c1e7da150f9c515fcc65871fcb219da08c3e578a

Reply:
7215290e447c5897cdef9fcf73bc6729f8998a64

Reply:
You didn't configure anything… also, why create a constant for that?

Reply:
e4b8b8e0248b1edaccb683e868720136f25352fa